### PR TITLE
Rewards improvements.

### PR DESCRIPTION
**Pull request by konrad04-1989**

### CODE REVIEW

File: app/controllers/rewards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class RewardsController < ApplicationController
+  # GET /rewards
+  def index
+    @rewards = Reward.all
+  end
+
+  # GET /rewards/1
+  def show
+    set_reward
```
Comment (ciejj) [inline app/controllers/rewards_controller.rb:11]: In general in RoR we do not use names like `set_foo`, `get_foo` - simply `foo`
but on the other hand, if we have a helper method, which is used only in one place I would consider not using it at all :) 
```suggestion
    @reward = Reward.find(params[:id])
```

File: spec/system/reward_system_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    driven_by(:rack_test)
+  end
+
+  let(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/MultipleExpectations
+
+  it 'enables me to show rewards list and show detailed reward page' do
```
Comment (ciejj) [inline spec/system/reward_system_spec.rb]: I would consider naming it a bit different:
```suggestion
  it 'displays rewards index and shows reward page' do
```

**Review by ciejj — APPROVED** [review]
Nice one, and really quick ;) Keep going!

File: app/controllers/rewards_controller.rb (hunk 1 above)
Comment (konrad04-1989) [reply app/controllers/rewards_controller.rb:11]: I wanted to keep the naming from previous controllers like set_kudo or set_company_value. To me it's a bit more readable for now. 

As for using a method in only one place... here I also wanted to keep the repeatability with the previous controllers. 

So the question is to change or leave it for now?

File: spec/system/reward_system_spec.rb (hunk 2 above)
Comment (konrad04-1989) [reply spec/system/reward_system_spec.rb]: Corrected!

**Review by oskarlakner — APPROVED** [review]

**Pull request merged**